app/models/concerns/users/info.rb

```diff
@@ -18,11 +18,21 @@ def member_at?(organization)
       organization_user_at(organization).present?
     end
 
-    def organizations_with_admin_access
+    def organizations_with_access
       if super_admin?
-        @organizations_with_admin_access ||= Organization.all
+        @organizations_with_access ||= Organization.all
       else
-        organizations.select { |org| admin_at?(org) }
+        organizations
+      end
+    end
+
+    def organizations_with_permission(permission)
```

Comment:
Cool! This all makes sense. Maybe, to make it more clear that organizations_with_permission is a list of organizations in which a particular permission is on, change the method name to: ```organizations_with_permission_enabled```